fontforge/parsettf.c

```diff
@@ -4765,8 +4778,12 @@ return;
 	    /* that's the amount of space left in the subtable and it must */
 	    /*  be filled with glyphIDs */
 	    if ( len<0 ) {
-		IError("This font has an illegal format 4 subtable with too little space for all the segments.\nThis error is not recoverable.\nBye" );
-		exit(1);
+		IError("This font has an illegal format 4 subtable with too little space for all the segments (%d).\nThis error is not recoverable.\n", rlen);
+		// exit(1);
+		// Exiting abruptly is not a desirable behavior.
+		// Even if continuing to load would result in a totally broken font,
+		// that is still better than closing completely.
```

Comment:
The only case where I can see this becoming a problem is if continuing can lead to another issue (e.g. some assumption about data being present when it's not) causing some sort of out of bounds access or the likes

Reply:
Yes. There is some risk of that. But any such crash could also be caused by a file otherwise misformed, and I'd like to trigger and to fix those.